src/components/vue-form-renderer.vue

```diff
@@ -143,6 +143,7 @@ export default {
       return child.errors();
     },
     isValid() {
```

Comment:
According to [JSDoc](https://en.wikipedia.org/wiki/JSDoc#JSDoc_tags) a function can be marked as deprecated this way:
```
/**
 * @deprecated Since version 2.4.4
 */
```

Reply:
changed

Reply:
docblock must go before function declaration, like: 
```
  /**
   * Calculates the circumference of the Circle.
   *
   * @deprecated since 1.1.0; use getCircumference instead
   * @return {number} The circumference of the circle.
   */
  calculateCircumference() {
    return 2 * Math.PI * this.radius
  }
```